tests/GetHelpTest.elm

```diff
@@ -22,13 +22,41 @@ suite =
                     |> Html.Styled.toUnstyled
                     |> Query.fromHtml
                     |> Query.contains [ Html.text (t GetHelpTitle) ]
-        , test "GetHelp view has 3 links" <|
+        , test "GetHelp view has 3 well formed call to action sections" <|
             \() ->
                 view
                     |> Html.Styled.toUnstyled
                     |> Query.fromHtml
-                    |> Query.findAll [ tag "a" ]
-                    |> Query.count (Expect.equal 3)
+                    -- Use the first title to ID the div containing all 3 sections
```

Comment:
Would be happy to start using `id="test-option-container"` or similar if it's easier.

Reply:
Yes. i am considering test id's but resisting for now. I think we should think about that when we review our strategy next week. I'll make a card for testing tasks.